portfolio/src/main/webapp/index.html

```diff
@@ -221,6 +221,7 @@ <h5 class="pinktext">HIT ME UP</h5>
                         </div>                      
                         <br />  
                         <button type="submit" class="btn btn-primary">Submit</button>
+                        <a id="myLink" href="https://nalemayehu-sps-summer21.appspot.com/viewemails.html" target="_blank">view all emails</a>
```

Comment:
We don't need to put the absolute address in href, it can be just "viewemails.html".
One advantage of using relative urls is, things will continue to work even if the app is hosted at a different domain/path.